src/interactivity/win32/windowio.cpp

```diff
@@ -123,7 +123,10 @@ bool HandleTerminalMouseEvent(const COORD cMousePosition,
     // Virtual terminal input mode
     if (IsInVirtualTerminalInputMode())
     {
-        fWasHandled = gci.GetActiveInputBuffer()->GetTerminalInput().HandleMouse(cMousePosition, uiButton, sModifierKeystate, sWheelDelta);
+        auto clampedPosition{ cMousePosition };

```

Comment:
Same here. What's the reason to clamp before a future maintainer get the bright idea to change it again.